Add note/warning about 'tenant' parameter (#115926)
Azure/azure-content: articles/active-directory/develop/v2-oauth2-client-creds-grant-flow.md

|     `state`     | A value that is included in the request that also is returned in the token response. It can be a string of any content that you want. The state is used to encode information about the user's state in the app before the authentication request occurred, such as the page or view they were on. |
| `admin_consent` | Set to **True**. |

> NOTE: Don't use tenant ID value of tenant parameter to authenticate or authorize users as the value can be updated and sent by anyone. It can occuer security incident for your application. 

Reviewer: ```suggestion
> [!NOTE] 
> Never use the **tenant ID** value of tenant parameter to authenticate or authorize users, as the value can be updated and sent by anyone. This can cause your application to be exposed to security incidents. 
```
